akka/src/main/scala/chatapp/Chat.scala

```diff
@@ -240,48 +240,96 @@ class Client(id: Long, directory: Directory, seed: Long) {
 }
 
 sealed trait DirMsg
-final case class DirLogin(id: Long)           extends DirMsg
-final case class DirBefriend()                extends DirMsg
-final case class DirLeft(client: ActorRef[Client]) extends DirMsg
-final case class DirPoke(factory: BehaviorFactory, accumulator: ActorRef[Accumulator]) extends DirMsg
-final case class DirDisconnect(poker: ActorRef[Poker]) extends DirMsg
+final case class DirLogin(id: Long) extends DirMsg
+final case class DirBefriend() extends DirMsg
+final case class DirLeft(client: ActorRef[ClientMsg]) extends DirMsg
+final case class DirPoke(factory: BehaviorFactory, accumulator: ActorRef[AccMsg]) extends DirMsg
+final case class DirDisconnect(poker: ActorRef[PokerMsg]) extends DirMsg
 
 class Directory(seed: Long, befriend: Int) {
-  val clients = mutable.ArrayBuffer[ActorRef[Client]]()
-  val random = SimpleRand(seed)
-  var poker: ActorRef[Poker] = null
```

Comment:
What's wrong with the name `poker`? And what's the `_` instead of the `null` here?

Reply:
In the variable scop the name poker is used twice. `_` and `null` can often used both, I changed it because my IDE marked it.

Reply:
Which IDE are you using? I am currently having the Metals plugin for VS code, to have Pony and the rest all in one thing.

Reply:
I us IntelliJ IDEA with scala plugin, only for akka